### DISCUSSION TITLE
runtime error

### DISCUSSION THREAD

**Discussion by TheFraed** [dmarx/video-killed-the-radio-star#125] [General]
Since yesterday, I run into the following error at https://colab.research.google.com/github/dmarx/video-killed-the-radio-star/blob/main/Video_Killed_The_Radio_Star_Defusion.ipynb#scrollTo=Sh514DGj_sua 

RuntimeError: Failed to import transformers.models.clip.feature_extraction_clip because of the following error 
(look up to see its traceback):
cannot import name 'builder' from 'google.protobuf.internal' 
(/usr/local/lib/python3.9/dist-packages/google/protobuf/internal/__init__.py)

**Comment by SchmitzAndrew** [comment]
This issue has been fixed with the latest release.
